pyproject.toml

```diff
@@ -84,6 +84,24 @@ omit = [
 ]
 
 
+# https://flit.pypa.io/en/latest/pyproject_toml.html#sdist-section
+# See also: https://github.com/pypa/flit/issues/565
+[tool.flit.sdist]
+include = []
+exclude = [
+    ".github/",
+    "docs/",
+    "tests/",
+    ".flake8",
+    ".gitignore",
+    ".pre-commit-config.yaml",
+    "CHANGELOG.md",
+    "Makefile",
+    "SECURITY.md",
+    "UPSTREAM_*.md",
```

Comment:
This might not to change when #245 is merged.

I see that you're keeping `README.md`. Wouldn't shipping `CHANGELOG.md` be a good idea too?

Reply:
> This might have to change when https://github.com/jenstroeger/python-package-template/pull/245 is merged.

Yup!

> I see that you're keeping `README.md`.

`flit` needed both `README.md` and `LICENSE.md` included so these two files can be added to the source dist.

> Wouldn't shipping `CHANGELOG.md` be a good idea too?

`flit` didn’t ask for it, so I haven’t added it yet. The changelog is also part of the metadata:

https://github.com/jenstroeger/python-package-template/blob/f8af74ddda359105680a06969de79382cf78f8ef/pyproject.toml#L58

I couldn’t find [mention of the changelog in sdist](https://docs.python.org/3/distutils/sourcedist.html#specifying-the-files-to-distribute), maybe @takluyver can shed some light on that?

Reply:
> This might not to change when #245 is merged.

Commit https://github.com/jenstroeger/python-package-template/pull/244/commits/adf530d938045c702b32e3fce478fa21cb05c52a.